src/server/ErrorHandler.js

```diff
@@ -6,3 +8,12 @@ export class APIError {
   }
 }
 
+export function handle404(req, res) {
+  if (!res.headersSent) {
+    res.status(404).send('Request > 404 - Page Not Found');
+    logger.error(`404 Not Found - ${req.method} - PATH : ${req.originalUrl} - ${new Date()}`);
+  }
+  if (res.statusCode === 200 || res.statusCode === 304) {
```

Comment:
Im not sure. Do you think it is better that remove this part?
Well, i think it is useless